docs/application/native/guides/multimedia/image-edit.md

```diff
@@ -82,214 +78,348 @@ To enable your application to use the image util functionality:
 
 4. Handle the transformation results in the `image_util_transform_completed_cb()` callback, which is invoked after the transformation is complete.
 
-5. After the transformation is complete, destroy the transformation handle using the `image_util_transform_destroy()` function:
+5. After the transformation is complete, destroy the transformation handle using `image_util_transform_destroy()`:
 
    ```
    ret = image_util_transform_destroy(handle);
    ```
 
-<a name="rotate"></a>
-## Rotating an Image
+<a name="rotate_image"></a>
+## Rotating Image
 
 To rotate an image:
 
-1. Create a transformation handle using the `image_util_transform_create()` function:
+1. Create a transformation handle using `image_util_transform_create()`:
 
    ```
-   transformation_h handle;
-   ret = image_util_transform_create(&handle);
+   transformation_h transform_h;
+   ret = image_util_transform_create(&transform_h);
    ```
 
-2. Set the amount of rotation using the `image_util_transform_set_rotation()` function:
+2. Set the amount of rotation using `image_util_transform_set_rotation()`:
 
    ```
-   ret = image_util_transform_set_rotation(handle, rotation);
+   ret = image_util_transform_set_rotation(transform_h, rotation);
    ```
 
    The possible values for the `rotation` parameter are defined in the `image_util_rotation_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__IMAGE__UTIL__TRANSFORM__MODULE.html#gad0682da8519f229944c9c5617b7a1107) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__IMAGE__UTIL__TRANSFORM__MODULE.html#gad0682da8519f229944c9c5617b7a1107) applications).
 
-3. Execute the transformation using the `image_util_transform_run()` function:
+3. Execute the transformation using `image_util_transform_run2()` or `image_util_transform_run2_async()`:
 
    ```
-   ret = image_util_transform_run(handle, src,
-                                  (image_util_transform_completed_cb)completed_callback,
-                                  user_data);
+   ret = image_util_transform_run2(transform_h, src_image, &dst_image);
    ```
 
    > **Note**
    >
-   > - The image format has no effect on the transformation.
-   > - If the color space is YUV, the target image width and height must be multiples of 8. This restriction does not apply to RGB images.If hardware acceleration is enabled, you can execute 2 more image transformations using the same transformation handle.
+   > - The image format has no effect on the transformation. If the color space is YUV, then the width and height of the target image must be multiples of 8. This restriction does not apply to the RGB images.
+
+4. Handle the transformation results in the `image_util_transform_completed2_cb()` callback, which is invoked after the transformation is complete.
```

Comment:
Suggest removing the and callback. Here in the `image_util_transform_completed2_cb()`, cb denotes callback. Therefore, suggest rephrasing to:
Handle the transformation results in `image_util_transform_completed2_cb()`, which is invoked after the transformation is complete.